portfolio/src/main/webapp/script.js

```diff
@@ -55,6 +55,50 @@ function pauseProjectChange() {
   clearTimeout(changeCurrProj);
 }
 
+google.charts.load('current', {'packages':['corechart']});
+google.charts.setOnLoadCallback(drawChart);
+
+/** Creates a chart and adds it to the page. */
+function drawChart() {
+  const data = new google.visualization.DataTable();
+  data.addColumn('string', 'Programming Languages');
+  data.addColumn('number', 'Count');
+  data.addRows([
+    ['Java', 10],
+    ['JavaScript', 8],
+    ['Python', 15]
+  ]);
+
```

Comment:
Can you add a comment here that describes the format of the array passed to addRows?